File: src/gestorAplicacion/personas/TipoDocumento.java
package gestorAplicacion.personas;

public enum TipoDocumento {
	CC("Cédula de Ciudadania"), CE("Cedula de Extranjería"), NIP("Numero de Identificación"),
	TI("Tarjeta de Identidad"), PAP("Pasaporte");

	public final String label;

	TipoDocumento(String label) {
		this.label = label;
	}
}

File: python/src/gestorAplicacion/personas/TipoDocumento.py
from enum import Enum

'''
# @author Alejandro Alvarez
# @author Alejandra Barrientos
# @author Mateo Alvarez
# @author Miguel Barrera'''

class TipoDocumento(Enum):
    CC = "Cédula de Ciudadanía"
    CE = "Cédula de Extranjería"
    NIP = "Número de Identificación"
    TI = "Tarjeta de Identidad"
    PAP = "Pasaporte"